Main/Service.java

```diff
@@ -0,0 +1,620 @@
+        System.out.println("9. Plaseaza comanda pentru un user dat (id) , catre un restaurant dat (id)");
+        System.out.println("10. Seteaza valoarea unei comenzi (id comanda citit de la tastatura) in functie de preparatele pe care le cuprinde, cat si de cantitatile lor");
+        System.out.println("11. Afisare meniu al unui restaurant al carui nume este dat");
+        System.out.println("12. Afisare ingrediente (inclusiv cantitatile) dintr-un produs al carui nume este dat");
+        System.out.println("13. Afisare soferi care folosesc un mijloc de transport dat (masina, scuter, bicicleta)");
+        System.out.println("14. Afisare restaurante dintr-un oras dat ca parametru");
+        System.out.println("15. Afisare toti userii in ordine crescatoare dupa varsta");
+        System.out.println("16. Modifica pretul unui produs dat");
+        System.out.println("17. Modifica adresa unui user al carui id este dat");
+        System.out.println("18. Afisare cel mai comandat produs");
+        System.out.println("0. Iesire");
+    }
+
+    public User createUser()
+    {
+        Scanner scanner;
+        scanner = new Scanner(System.in);
+        System.out.println("Introduceti numele userului: ");
+        String lastName = scanner.nextLine();
+        System.out.println("Introduceti prenumele userului: ");
+        String firstName = scanner.nextLine();
+        System.out.println("Introduceti email-ul userului: ");
+        String email = scanner.nextLine();
+        System.out.println("Introduceti numarul de telefon al userului: ");
+        String phoneNo = scanner.nextLine();
+        System.out.println("Introduceti varsta userului: ");
+        Integer age = scanner.nextInt();
+        scanner.nextLine();
+        System.out.println("Introduceti id-ul user-ului: ");
+        Integer id = scanner.nextInt();
+        scanner.nextLine();
+        System.out.println("Introduceti username: ");
+        String username = scanner.nextLine();
+        Address a = createAddress();
+        User u = new User(lastName, firstName, email, phoneNo, age, id, username, a);
+        return u;
+    }
+
+    public Transport createTransport()
+    {
+        Scanner scanner;
+        scanner = new Scanner(System.in);
+        System.out.println("Ce mijloc de transport doriti sa creati? (OPTIUNILE SUNT: masina/scuter/bicicleta) ");
+        String opt = scanner.nextLine();
+        String optiune = opt.replace("\n", "");
+        while (true)
+        {
+            if (optiune.equalsIgnoreCase("masina") || optiune.equals("scuter") || optiune.equalsIgnoreCase("bicicleta"))
+                break;
+            else
+            {
+                System.out.println("Nu ati introdus o optiune valida, incercati din nou!");
+                System.out.println("Reintroduceti optiunea: ");
+                opt = scanner.nextLine();
+                optiune = opt.replace("\n", "");
+            }

```

Comment:
maybe you consider throwing an exception if the user does not chose a right option or let him insert 2 or 3 times before this